examples/example.yaml.in

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 The Cockroach Authors
```

Comment:
Same comment as above, splitting the template files from the rendered ones.

Reply:
Done